GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+# Eiffel Community Governance
+
+Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability.
+Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
+
+Governance of the Eiffel Community is structured according to this document.
+This structure in intended to provide a healthy, sustainable and attractive community for Eiffel development and collaboration.
+
+The Eiffel community proactively adopts and follows best practices employed by successful open source communities while maintaining a governance agreeable to it's members.
+
+# Governance
+
+Eiffel is a community of projects working on building CI/CD Pipelines using Eiffel Protocol.
```

Comment:
I would like to not restrict this to CI/CD pipelines - we are seeing interest in for example Statement of Verification for requirements (using Issue-events). ".... SW Pipelines using Eiffel protocol for CI/CD and other practices". 

This is related to the vision discussion in our summit.

Reply:
Makes sense.

I kind of took this from the Eiffel descriptionwhich is in the README files in all the repos.

"Eiffel is a protocol for technology agnostic machine-to-machine communication in **continuous integration and delivery pipelines**, aimed at securing scalability, flexibility and traceability."

I can amend this change based on your comment but we should perhaps first think about how to reword what Eiffel is. (or I can remove this sentence from the governance document to limit impacts of changes in vision/mission)